docs/topics/Permissions.md

```diff
@@ -161,14 +162,10 @@ There may be other cases in which certain permissions implicitly deny or allow o
 
 ## Inherited Permissions (Threads)
 
-Threads inherit permissions from the parent channel (the channel they were created in).
+Threads inherit permissions from the parent channel (the channel they were created in), with one exception: The `SEND_MESSAGES` permission is not inherited, users must have `SEND_MESSAGES_IN_THREADS` to send a message in a thread, which allows for users to participate in threads in places like announcement channels.
 
 Users must have the `VIEW_CHANNEL` permission to view _any_ threads in the channel, even if they are directly mentioned or added to the thread.
 
-Users can **create** a thread if they have _both_ the `SEND_MESSAGES` permission and the appropriate threads permission (`USE_PUBLIC_THREADS` for public threads, `USE_PRIVATE_THREADS` for private threads) on the parent channel.
```

Comment:
I'm assuming users will still need Use X Threads to create the thread right?

Reply:
I'm not sure how to answer that question, since the point of this PR is to "remove" those permissions :)
![image](https://user-images.githubusercontent.com/126578/129783092-d258b80c-c793-41d5-84a1-beae3ba3618c.png)

Reply:
Sorry, that's me being sleepy :P